code/modules/mob/living/carbon/human/species/station/getmorean/getmorean.dm

```diff
@@ -0,0 +1,115 @@
+/datum/species/getmorean
+	name = "Getmorean"
+	hide_name = FALSE
+	short_name = "get"
+	name_plural = "Getmoreans"
+	bodytype = "Getmorean"
+	age_max = 20
+	economic_modifier = 4 // Getmorean don't get paid much money at all, as they're technically a hired corperate entity.
+	icobase = 'icons/mob/human_races/getmorean/r_getmorean.dmi'
+	deform = 'icons/mob/human_races/getmorean/r_def_getmorean.dmi'
+	preview_icon = 'icons/mob/human_races/getmorean/getmorean_preview.dmi'
+
+	default_genders = list(MALE)
+
+	// No need to eat or drink. They are drink.
+	max_hydration_factor = -1
+	max_nutrition_factor = -1
+
+	flesh_color = "#333333"
+
+	mob_size = MOB_SMALL
+
+	hud_type = /datum/hud_data/getmorean // Doesn't have a head. No head slots.
+
+	brute_mod =     3.0                  // Physical damage multiplier.
+	burn_mod =      0.3                  // Burn damage multiplier.
```

Comment:
This seems excessive and will most likely need to be tweaked a bit

Reply:
Solo balance is hard, but I feel it's plenty considering their other resistances.